### DISCUSSION TITLE
FireTV App crashes on older FireTV Sticks

### DISCUSSION THREAD

**Discussion by RocketAlf** [GerryDazoo/SlingerPlayer#6] [Q&A]
The SlingerPlayer for the Fire TV Stick unfortunately does not run on the older sticks. I have a FireTV Stick (2nd Gen) SW version "FireOS 5.2.9.4". The app can be installed on the stick. When starting the app it crashes shortly after the opening screen.

Is there any hope that this could be fixed in one of the next updates?

**Comment by GerryDazoo** [comment]
Since I have no way of knowing what might be failing there is no hope that this will get fixed or even if it is fixable.

**Comment by RocketAlf** [comment]
I have now purchased a newer FireTV Stick :-/. SlingerPlayer for FireTV can be installed and runs problem-free :-).

@Gerry: I just haven't figured out how to open the remote control page. Or does that not work at all on the FirteTV Stick?

@Anyone who is interested. On the older FireTV Stick, SlingerPlayer installs, but the program crashes. BUT ...it is also possible to watch the streams from the SLingerServer using  e.g. VLC for FireTV (easy way),  **OR**  ...even on the older FireTV Stick Amazon Alexa is integrated (a bit more complex, but easy to add if you already use Monocle-Cam with the Echo Show). 
That is, the function to call (surveillance) cameras also works by voice command via the FireTV remote (press the microphone button), e.g. "Alexa, show CAM1", just like on the Echo Show.  How to connect SlingerServer (just like a surveillance camera) to the Echo Show or also to the FireTV Stick via voice command I explained in the discussion "bring the Slingbox to the Echo Show #505" (https://github.com/GerryDazoo/Slinger/discussions/505).
The only drawback for both is that the remote control is not integrated and must be implemented separately.

**Reply by GerryDazoo** [reply]
There is no integrated remote on the Firestick version. It's basically unusable without some sort of mouse to navigate with. I just use a browser on my phone for remote control.

**Reply by martysm** [reply]
I just got the fire stick up and running at the summer place, over a persistent Wireguard tunnel with the house where the slingbox and slinger live.  It works wonderfully even with the asymmetric (800/12) Comcast connection back there.  I just happened to pop in here to see what people do for a remote on the fire stick.  Thanks for answering in advance.  That's what I figured bc I just fired up the app on my phone and used that, for expediency.  A browser saved to the desktop and placed next to the slingplayer icon will work just fine.

Thanks again Gerry!

ps- You know, it's funny.  The SB350 was a gift to my wife, 10-15 years ago, because she's a night-time insomniac Lifetime TV watcher and there was no cable provider down here at the time.  She is also not an android user, so she has expressed great annoyance as I have taken over "her" Slingbox.  Whenever she bitches about it I just say " You COULD get a reasonably priced phone or tablet and it will work for you, too.  Maybe ask Tim Cook to hook you up.", or something similarly snarky. (:-)))))
